### PR TITLE
feat(spanner): add standalone samples for NUMERIC type

### PR DESCRIPTION
**Pull request by larkee**

### CODE REVIEW

File: spanner/spanner.php [hunk 1]
```diff
@@ -303,6 +315,18 @@
     })
 );
 
+// Update data with timestamp column command
```
Comment (thiagotnunes) [inline spanner/spanner.php:318]: I think this comment should say "Update data with numeric column command"

Comment (larkee) [reply spanner/spanner.php]: Good spotting! Fixed :+1:

**Review by thiagotnunes — APPROVED** [review]

File: spanner/src/update_data_with_numeric_column.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * @param string $instanceId The Spanner instance ID.
+ * @param string $databaseId The Spanner database ID.
+ */
+function update_data_with_numeric_column($instanceId, $databaseId)
+{
+    $spanner = new SpannerClient();
+    $instance = $spanner->instance($instanceId);
+    $database = $instance->database($databaseId);
+
+    $operation = $database->transaction(['singleUse' => true])
```
Comment (bshaffer) [inline spanner/src/update_data_with_numeric_column.php:49]: According to the documentatioon, this returns a `Google\Cloud\Core\Timestamp` object and not an `$operation`. Also this parameter is never used. So I'd recommend removing it.

Comment (larkee) [reply spanner/src/update_data_with_numeric_column.php]: Done. As a note, this template exists in other samples. I'll make a separate PR to correct this in those samples.

**Review by bshaffer — APPROVED** [review]

**Comment by larkee** [comment]
This PR cannot be merged until GoogleCloudPlatform/php-docs-samples/pull/1184 is merged to skip the failing test.

**Comment by skuruppu** [comment]
@bshaffer, if you can help us merge this in, that would be awesome. The tests now pass :)

**Pull request merged**